crates/ruma-common/src/api.rs

```diff
@@ -53,10 +46,10 @@ use ruma_identifiers::UserId;
 /// }
 /// ```
 ///
-/// This will generate a `ruma_api::Metadata` value to be used for the `ruma_api::Endpoint`'s
-/// associated constant, single `Request` and `Response` structs, and the necessary trait
-/// implementations to convert the request into a `http::Request` and to create a response from
-/// a `http::Response` and vice versa.
+/// This will generate a `ruma_common::api::Metadata` value to be used for the
+/// `ruma_common::api::Endpoint`'s associated constant, single `Request` and `Response`
+/// structs, and the necessary trait implementations to convert the request into a
+/// `http::Request` and to create a response from a `http::Response` and vice versa.
```

Comment:
Would be nice to remove the `ruma_common::api::` prefixes here and add intradoc links.

Reply:
Actually `Endpoint` doesn't exist (anymore?). Should I just use it as a regular word?

Reply:
Hah, yeah that was replaced by `IncomingRequest` and `OutgoingRequest`. Let's mention those, e.g. "for the associated constants of …".